packages/stream_chat_flutter/lib/src/channel_avatar.dart

```diff
@@ -108,7 +107,7 @@ class ChannelAvatar extends StatelessWidget {
                   imageUrl: channelImage,
                   errorWidget: (_, __, ___) => Center(
                     child: Text(
-                      extraData['name']?[0] ?? '',
+                      channel.name?[0] ?? '',
```

Comment:
I'm not sure if this will work? Will channel.name automatically update? Or are we fine with displaying an outdated name, and just getting updates for the image?

Reply:
`channel.name` is a getter so it will always give back the latest name

Reply:
Will it though?  Does the `Channel` get updated when the channel state changes? Or is only new values emitted from the different streams?